docs/computing/containers/creating.md

```diff
@@ -76,3 +76,54 @@ Below is a table of common docker base images and whether installing simple pack
 |ubuntu|16.04-22.04|yes|
 
 Some issues related to, for example, glibc, fakeroot, file permissions and old remote repos are often difficult to solve, so trying out a few different base images can be a good idea before spending a lot of time debugging.
+
+## Using GPU from containers in interactive sessions in Puhti
+
+### Running existing images
+
+To run programs in [accelerated visualization](https://docs.csc.fi/computing/webinterface/accelerated-visualization/) that [use GPU](https://apptainer.org/docs/user/latest/gpu.html), use the `--nv` flag when starting the container: `apptainer run --nv /path_to_image/image.sif`. To use the graphical display with [VirtualGL](https://virtualgl.org/), a few environment variables have to be set as well. In the base images provided for GPU usage by CSC, these are set automatically when the container is started. Note that if you run `apptainer shell` instead of `apptainer run`, `%runscript` is not executed and necessary environment variables for vgl are not set, you then have to set them manually, see [base image definition files](https://github.com/CSCfi/singularity-recipes/tree/main/visualization) for details.
```

Comment:
```suggestion
To run programs in [Accelerated Visualization](../webinterface/accelerated-visualization/) that [use GPU](https://apptainer.org/docs/user/latest/gpu.html), use the `--nv` flag when starting the container: `apptainer run --nv /path_to_image/image.sif`. To use the graphical display with [VirtualGL](https://virtualgl.org/), a few environment variables have to be set as well. In the base images provided for GPU usage by CSC, these are set automatically when the container is started. Note that if you run `apptainer shell` instead of `apptainer run`, `%runscript` is not executed and necessary environment variables for vgl are not set, you then have to set them manually, see [base image definition files](https://github.com/CSCfi/singularity-recipes/tree/main/visualization) for details.
```

Reply:
Could you also add a link from the Accelerated Visualization page (e.g. under "Using the app") to this "Using GPU from containers in interactive sessions in Puhti" section?